core/src/main/java/org/elasticsearch/node/InternalSettingsPreparer.java

```diff
@@ -193,6 +194,10 @@ private static String promptForValue(String key, Terminal terminal, boolean secr
                 + "]. prompting for property values is only supported when running elasticsearch in the foreground");
         }
 
+        terminal.println(Terminal.Verbosity.SILENT,
+            "Prompting for property values is deprecated in " + Version.CURRENT
```

Comment:
I think this is a little misleading. Today prompting can be used on any setting, but not every setting is eligible for secure settings.

Reply:
Oh i was not aware of this! ty for this info. im also bumping this to 6.3